src/apps/hedgey/shortcuts.ts

```diff
@@ -3,15 +3,16 @@ import { celo } from 'viem/chains'
 import { ShortcutsHook } from '../../types/shortcuts'
 import { getHedgeyPlanNfts } from './nfts'
 import { tokenVestingPlansAbi } from './abis/token-vesting-plans'
+import { NetworkId } from '../../api/networkId'
 
 const client = createPublicClient({
   chain: celo,
   transport: http(),
 })
 
 const hook: ShortcutsHook = {
-  async getShortcutDefinitions(network?: string, address?: string) {
-    if (network !== 'celo' || !address) {
```

Comment:
@jeanregisser should the other shortcuts also check if the network id is defined and equal to Celo if that's the only network the dapp is on? I tried adding the check everywhere, but found that it broke some e2e tests in `runtime/getShortcuts.e2e.ts` that explicitly pass in "undefined" for the network. The tests seem to indicate that missing network id's are supposed to be accepted in general. In fact, the network id is ignored for the most part (except for locked celo), meaning you could query for ethereum shortcuts and get a bunch of celo results. 

I tried out using the network id parameter to compare against arrays of supported networks for each dapp in commit `4a94b15a34817f36e68cf6aaa89691475724fbf0`, but ran up against the e2e test failure. 

Lmk what the intent here and how you think we should proceed

Reply:
it looks like the 'network' parameter was required in the api, which makes me think maybe the e2e test enforcing the acceptance of undefined network parameters was not testing something that's important to preserve, but lmk if I'm mistaken about that

Reply:
ok, since no shortcuts are multichain right now I'm ok with leaving it for when that functionality is added then